x/fbridge/keeper/grpc_query.go

```diff
@@ -76,22 +85,94 @@ func (k Keeper) Commitments(ctx context.Context, request *types.QueryCommitments
 	panic("implement me")
 }
 
-func (k Keeper) Guardians(ctx context.Context, request *types.QueryGuardiansRequest) (*types.QueryGuardiansResponse, error) {
-	panic("implement me")
+func (k Keeper) Members(goCtx context.Context, req *types.QueryMembersRequest) (*types.QueryMembersResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	role := types.QueryParamToRole[req.Role]
+	if role == 0 {
+		return nil, status.Error(codes.InvalidArgument, "invalid role")
+	}
+
+	members := make([]string, 0)
+	roles := k.GetRolePairs(ctx)
+	for _, pair := range roles {
+		if pair.Role == role {
+			members = append(members, pair.Address)
+		}
+	}
+
+	return &types.QueryMembersResponse{Members: members}, nil
 }
 
-func (k Keeper) Operators(ctx context.Context, request *types.QueryOperatorsRequest) (*types.QueryOperatorsResponse, error) {
-	panic("implement me")
+func (k Keeper) Member(goCtx context.Context, req *types.QueryMemberRequest) (*types.QueryMemberResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	r := k.GetRole(ctx, sdk.MustAccAddressFromBech32(req.Address))
```

Comment:
`Proposal()` and `Vote()` return an error if no entry is found. We may provide the same experience here.

Reply:
It will return `unspecified`. Does it seem weird?

Reply:
It also makes sense so you don't need to change the logic. My point is, in case of `Vote()` there is also similar alternative, returning unspecified vote option, but it throws error.